fe/fe-core/src/main/java/org/apache/doris/fs/obj/HdfsStorage.java

```diff
@@ -15,9 +15,11 @@
 // specific language governing permissions and limitations
 // under the License.
 
-package org.apache.doris.backup;
```

Comment:
Why moving `hdfs` to `fs/obj`? hdfs is not a object storage system

Reply:
I will remove it in PR2 as it is a deprecated class.